mne_bids/copyfiles.py

```diff
@@ -19,8 +19,9 @@
 import re
 
 import shutil as sh
+from datetime import datetime as dt
```

Comment:
let's try to not use abbreviations beyond the most common ones, because it can make the code harder to read, rather than shorter+easier.

```suggestion
from datetime import datetime
```